Makefile

```diff
@@ -18,6 +19,20 @@ setup:
 	vagrant up
 	vagrant ssh-config > ssh_config
 
+
+batch:
+	$(foreach GROUP,$(GROUPS),GROUP=$(GROUP) $(MAKE) run-group || exit 1;)
```

Comment:
We have a mix of `${}` and `$()` styles. Let's converge on `${}`.

Reply:
I named all variables with ${VAR}.